cmake/ExternalProjects.cmake

```diff
@@ -15,8 +15,8 @@ FetchContent_Declare(wavm_ext
 )
 
 FetchContent_Declare(wamr_ext
-    GIT_REPOSITORY "https://github.com/J-Heinemann/wasm-micro-runtime"
-    GIT_TAG "5a3db31c096ec4d8a7b8b639aeaecd3fdda06b30"
+    GIT_REPOSITORY "https://github.com/faasm/wasm-micro-runtime"
```

Comment:
Both references point to the same commit.

Reply:
I thought that we stopped using branch names because CMake wouldn't update properly? Should the `faasm` here not be the commit hash? If the problem with branch names is fixed, can we stop using commit hashes elsewhere?

Reply:
You are right. The problem with pining to branch names is that CMake has no way of telling if it has to update the dependency or not (i.e. there are newer commits in the branch). Through commit hashes it does.